src/me/StevenLawson/TotalFreedomMod/Commands/Command_spectator.java

```diff
@@ -0,0 +1,73 @@
+package me.StevenLawson.TotalFreedomMod.Commands;
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.OP, source = SourceType.BOTH)
```

Comment:
I was under the impression that Spectator mode would be for admins only, not sure if @JeromSar agrees with that, but I definitely see that as the better way to go.